_posts/2016-06-24-myeong-intro.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title:  "Introducing DSSG Fellow Myeong Lee"
+date:   2016-06-24
+author: Myeong Lee
+---
+
+<img src="http://myeonglee.com/sites/default/files/myeong_2.jpg" width="200" height="200" align="left" style="margin: 0 10px 10px 0"> I am a 2nd-year doctoral student at the University of Maryland, College Park studying information science (a.k.a., Maryland's iSchool). 
+
+Specifically, I am interested in understanding how socio-technical factors affect various kinds of communities and organizations, and in designing social computing systems to solve community problems. This academic field is usually called Community/Urban Informatics, and it stretches to the fields of human-computer interaction (HCI) and computational social science depending on the focus of a study. I often use computational approaches to understanding the dynamics of local communities, organizations, and individuals' behavior based on various kinds of data (e.g., geo-tagged social media data), and want to further theorize how socio-technical factors shape the overall local information landscape in cities in the future. 
+
+UW's DSSG this summer attracted me a lot, because the topics and skillsets that the program covered were perfectly fit to my interests and research directions. Especially, the project that I am working on, CrowdCensus, is an urban/social computing project that makes use of various kinds of geo-tagged data such as cellphone detail records (CDR) and social media data to predict socio-economic status of urban regions. Through this project, I am willing to build an easy-to-use model and system that can help policy-makers and urban planners, but also understand the data themselves so to better conceptualize geo-tagged data with a theoretical framework. I am really glad that I can work with brilliant people from diverse academic disciplines. 
```

Comment:
The word "willing" might not be the best word here. How about "hoping"? Or "working"?